public/docs/ts/latest/guide/setup.jade

```diff
@@ -0,0 +1,121 @@
+  1. run it
+
+  ## What's different?
+
+  The setup delivers the same application as the [QuickStart](../quickstart.html "Angular QuickStart"),
+  a fact you can confirm with the <live-example></live-example>.
+  
+  But there are _lot more_ files in the project folder on your machine. 
+  Ignore all but three of them for now and 
+  [learn about the others later](setup-systemjs-anatomy.html "Setup Anatomy").
+  
+  Focus on these three TypeScript (`.ts`) files in the **`/app`** folder.
+
+.filetree
+  .file app
+  .children
+    .file app.component.ts
+    .file app.module.ts
+    .file main.ts
+
++makeTabs(`
+    setup/ts/app/app.component.ts,
+    setup/ts/app/app.module.ts,
+    setup/ts/app/main.ts
+  `, '', `
+    app/app.component.ts,
+    app/app.module.ts,
+    app/main.ts
+  `)(format='.')
+:marked
+  The guides and cookbooks in these page expect you to have re-factored the [QuickStart](../quickstart.html "Angular QuickStart") sample
+  into these three files. It's wise to do so. 
+  Each file has a distinct purpose and evolves independently as the application grows.
+
+style td, th {vertical-align: top}
+table(width="100%")
+  col(width="20%")
+  col(width="80%")
+  tr
+    th File
+    th Purpose
+  tr
+    td <code>app.component.ts</code>
+    td
+      :marked
+        The `AppComponent` currently displays the "Hello Angular" header.
+        It is the root of what will become a tree of nested components. 
+  tr
+    td <code>app.module.ts</code>
+    td
+      :marked
+        The `AppModule` tells Angular how to assemble the application.
+        There will soon be more than the `AppComponent` to talk about. 
+  tr
+    td <code>main.ts</code>
+    td
+      :marked
+        `Main` holds instructions for dynamically bootstrapping the application in the browser.
+        That's the only choice when running in the live-coding browser environment.
+        You'll learn about other compiling and deployment options later in the documentation.
+
+#install-node
+.l-main-section
+:marked
+  ## Prerequisites: _node_ and _npm_
+
+    Node.js and npm are essential tools in modern web development. Angular developers need them too.
+    
+    <a href="https://docs.npmjs.com/getting-started/installing-node" target="_blank" title="Installing Node.js and updating npm">
+    Get them now</a> if they're not already installed on your machine.
+ 
+    **Verify that you are running node `v4.x.x` or higher and npm `3.x.x` or higher**
+    by running the commands `node -v` and `npm -v` in a terminal/console window.
+    Older versions produce errors.
+
+    We recommend [nvm](https://github.com/creationix/nvm) for managing multiple versions of node and npm. You may need [nvm](https://github.com/creationix/nvm) if you already have projects running on your machine that use other versions of node and npm.
+  
+.l-sub-section
+  :marked
+    ### Next step
+    [The _Tour of Heroes_ tutorial](../tutorial)

```

Comment:
We trigger a "next step" automagically for the basic guides so this will trigger two next steps. No bueno.